packages/loot-core/src/platform/client/fetch/__mocks__/index.web.js

```diff
@@ -0,0 +1,68 @@
+let listeners = new Map();
```

Comment:
This file is a copy&paste from `packages/loot-core/src/platform/client/fetch/index.testing.js` with no changes. It's really just a file move.